zh_cn.json

```
@@ -0,0 +1,139 @@
+{
+  "fluid_type.brewinandchewin.honey_type": "蜂蜜",
+  "fluid_type.brewinandchewin.beer_type": "啤酒",
+  "fluid_type.brewinandchewin.vodka_type": "伏特加",
+  "fluid_type.brewinandchewin.mead_type": "蜂蜜酒",
+  "fluid_type.brewinandchewin.egg_grog_type": "鸡蛋格罗格",
+  "fluid_type.brewinandchewin.strongroot_ale_type": "强根麦芽酒",
+  "fluid_type.brewinandchewin.rice_wine_type": "米酒",
+  "fluid_type.brewinandchewin.glittering_grenadine_type": "发光果浆",
+  "fluid_type.brewinandchewin.steel_toe_stout_type": "钢趾黑啤",
+  "fluid_type.brewinandchewin.dread_nog_type": "惧怖蛋奶酒",
+  "fluid_type.brewinandchewin.kombucha_type": "康普茶",
+  "fluid_type.brewinandchewin.saccharine_rum_type": "甜果朗姆",
+  "fluid_type.brewinandchewin.pale_jane_type": "苍白珍妮",
+  "fluid_type.brewinandchewin.salty_folly_type": "咸味讽醴",
+  "fluid_type.brewinandchewin.bloody_mary_type": "血腥玛丽",
+  "fluid_type.brewinandchewin.red_rum_type": "红色朗姆",
+  "fluid_type.brewinandchewin.withering_dross_type": "凋零糟粕",
+  "fluid_type.brewinandchewin.flaxen_cheese_type": "淡黄奶酪",
+  "fluid_type.brewinandchewin.scarlet_cheese_type": "绯红奶酪",
+
+  "block.brewinandchewin.keg": "发酵桶",
+  "block.brewinandchewin.heating_cask": "加热桶",
+  "block.brewinandchewin.ice_crate": "箱装冰",
+  "block.brewinandchewin.coaster": "托垫",
+
+  "item.brewinandchewin.tankard": "酒杯",
+  "item.brewinandchewin.beer": "啤酒",
+  "item.brewinandchewin.vodka": "伏特加",
+  "item.brewinandchewin.mead": "蜂蜜酒",
+  "item.brewinandchewin.rice_wine": "米酒",
+  "item.brewinandchewin.egg_grog": "鸡蛋格罗格",
+  "item.brewinandchewin.strongroot_ale": "强根麦芽酒",
+  "item.brewinandchewin.saccharine_rum": "甜果朗姆",
+  "item.brewinandchewin.pale_jane": "苍白珍妮",
+  "item.brewinandchewin.salty_folly": "咸味讽醴",
+  "item.brewinandchewin.steel_toe_stout": "钢趾黑啤",
+  "item.brewinandchewin.glittering_grenadine": "发光果浆",
+  "item.brewinandchewin.bloody_mary": "血腥玛丽",
+  "item.brewinandchewin.red_rum": "红色朗姆",
+  "item.brewinandchewin.withering_dross": "凋零糟粕",
+  "item.brewinandchewin.kimchi": "泡菜",
+  "item.brewinandchewin.jerky": "肉干",
+  "item.brewinandchewin.pickled_pickles": "腌泡菜",
+  "item.brewinandchewin.kippers": "腌鱼",
+  "item.brewinandchewin.cocoa_fudge": "可可软糖",
+  "item.brewinandchewin.vegetable_omelet": "蔬菜欧姆蛋",
+  "item.brewinandchewin.cheesy_pasta": "奶酪意面",
+  "item.brewinandchewin.creamy_onion_soup": "奶油洋葱汤",
+  "item.brewinandchewin.scarlet_pierogi": "绯红波兰饺",
```

```suggestion
  "item.brewinandchewin.scarlet_pierogi": "绯红波兰饺子",
```
在国内搜索引擎搜索，目测“波兰饺子”的词频远高于“波兰饺”。